website/docs/cloud-docs/integrations/kubernetes/k8s-ops-v2-migration.mdx

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Terraform Cloud Kubernetes Operator v2 Migration Guide
+description: >-
+  Upgrade the Terraform Cloud Kubernetes Operator from version 1 to version 2.
+tfc_only: false
+---
+
+# Terraform Cloud Kubernetes Operator v2 Migration Guide
+
+To upgrade the Terraform Cloud Kubernetes Operator from version 1 to version 2, there is a one-time process that you need to complete. This process will upgrade the operator to the newest version and migrate your custom resources. 
+
+## Prepare for the upgrade
+
+Before you migrate to Terraform Cloud Kubernetes Operator v2, you must first update v1 of the operator to the latest version, including the custom resource definitions.
+
+```shell-session
+$ helm upgrade --namespace ${RELEASE_NAMESPACE} ${RELEASE_NAME} hashicorp/terraform
+```
+
+Next, backup the workspace resources.
+
+```shell-session
+$ kubectl get workspace --all-namespaces -o yaml > backup_tfc_operator_v1.yaml
+```
+
+Finally, we suggest that you set the `applyMethod` of each workspace to `manual`. In version 2 of the Terraform Cloud Kubernetes Operator this is the default value, however explicitly configuring it with the value provides extra protection from unintended automatic applies during the upgrade process.
+
+```shell-session
+kubectl patch workspace <WORKSPACE_NAME> --type=merge --patch '{"spec": {"applyMethod": "manual"}}'
+```
+## Upgrade the operator
+
+Clone the Terraform Cloud Kubernetes Operator repository.
+
+```shell-session
+$ git clone https://github.com/hashicorp/terraform-cloud-operator.git
+```
+
+Change to the migration directory.
+
+```shell-session
+$ cd terraform-cloud-operator/docs/migration/crds
+```
+
+View the changes that **patch A** will apply to the workspace CRD.
+
+```shell-session
+$ kubectl diff -f workspaces_patch_a.yaml
+```
+
+Patch the workspace CRD with **patch A**. This patch adds `app.terraform.io/v1alpha2` support, but excludes `.status.runStatus` because it has a different format in `app.terraform.io/v1alpha1` and causes JSON unmarshalling issues.
+
+```shell-session
+$ kubectl patch crd workspaces.app.terraform.io --patch-file workspaces_patch_a.yaml
+```
+
+Follow the [installation guide](https://github.com/hashicorp/terraform-cloud-operator/blob/main/docs/installation.md) to install v2 of the operator.
```

Comment:
So we're supposed to leave this guide and follow the steps in that one? That page has a few different methods to install - should I follow the `install` command or the `upgrade` command? I think this needs a little more explicit guidance.

Reply:
Pulled the direct install command from the guide, need to verify with eng that this is the intended use.